nava/functions.py

```diff
@@ -53,137 +82,123 @@ def quoter(sound_path, *args, **kwargs):
     return quoter
 
 
-def cleanup_processes():
+def __play_win(sound_path, async_mode=False):
     """
-    Cleanup undead play processes after module exit.
+    Play sound in Windows.
 
-    :return: None
+    :param sound_path: sound path
+    :type sound_path: str
+    :param async_mode: async mode flag
+    :type async_mode: bool
+    :return: None or sound id
     """
-    for proc in play_processes:
-        proc.kill()
-        proc.terminate()
+    import winsound
+    play_flags = winsound.SND_FILENAME | (async_mode & winsound.SND_ASYNC)
+
+    if async_mode:
+        sound_thread = NavaThread(target=__play_win_flags, args=(sound_path, play_flags), daemon=True)
```

Comment:
It's too long. Can you make it multiline?

Reply:
fixed --> https://github.com/openscilab/nava/pull/29/commits/b0f52fcbda301d406a0ba456c779ac3575459f07